<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-system">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabAutoLogin">
      <attribute name="title">
       <string>Autologin</string>
      </attribute>
      <layout class="QGridLayout" name="layAutoLogin">
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <property name="spacing">
        <number>12</number>
       </property>
       <item row="4" column="0">
        <widget class="QLabel" name="lblUsername">
         <property name="text">
          <string>Username for autologin session</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="txtUsername"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="chkRelogin">
         <property name="text">
          <string>Whether sddm should automatically log back into sessions when they exit</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="lblRelogin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Name of session file for autologin session (if empty try last logged in)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QComboBox" name="cbxSessionFile"/>
       </item>
       <item row="5" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="layGeneral">
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <property name="spacing">
        <number>12</number>
       </property>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="txtInputMethod"/>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="txtHaltCommand"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblRebootCommand">
         <property name="text">
          <string>Reboot command</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="lblNumlockDesc">
         <property name="text">
          <string>If property is set to none, numlock won't be changed
NOTE: Currently ignored if autologin is enabled.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="txtRebootCommand"/>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="cbxNumlock">
         <item>
          <property name="text">
           <string notr="true">none</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">on</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">off</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="lblNumlock">
         <property name="text">
          <string>Initial NumLock state. Can be on, off or none.</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblInputMethod">
         <property name="text">
          <string>Input method module</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblHaltCommand">
         <property name="text">
          <string>Halt command</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <spacer name="vspGeneral">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTheme">
      <attribute name="title">
       <string>Theme</string>
      </attribute>
      <layout class="QGridLayout" name="layTheme">
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <property name="spacing">
        <number>12</number>
       </property>
       <item row="6" column="3" colspan="2">
        <widget class="QLineEdit" name="txtThemeDir"/>
       </item>
       <item row="6" column="5">
        <widget class="QToolButton" name="tbnThemeDir">
         <property name="icon">
          <iconset theme="folder-open">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="QToolButton" name="tbnFacesDir">
         <property name="icon">
          <iconset theme="folder-open">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="3" colspan="2">
        <widget class="QLineEdit" name="txtFacesDir"/>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QLabel" name="lblThemeDir">
         <property name="text">
          <string>Theme directory path</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="5">
        <widget class="QLabel" name="lblAvatarLimitDesc">
         <property name="text">
          <string>above which avatars are disabled
unless explicitly enabled with EnableAvatars</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblCurrentTheme">
         <property name="text">
          <string>Current theme name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="lblCursorTheme">
         <property name="text">
          <string>Cursor theme used in the greeter</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="lblFacesDir">
         <property name="text">
          <string>Global directory for user avatars</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3" colspan="3">
        <widget class="QSpinBox" name="sbxAvatarsThreshold"/>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="lblAvatarsThreshold">
         <property name="text">
          <string>Number of users to use as threshold</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" colspan="3">
        <widget class="QLineEdit" name="txtCursorTheme"/>
       </item>
       <item row="0" column="3" colspan="3">
        <widget class="QComboBox" name="cbxCurrentTheme"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pbnThemePreview">
         <property name="text">
          <string>Preview</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="6">
        <widget class="QCheckBox" name="chkEnableAvatars">
         <property name="text">
          <string>Enable display of custom user avatars</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="6">
        <widget class="QLabel" name="lblAvatarDirDesc">
         <property name="text">
          <string>The files should be named &lt;username&gt;.face.icon</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabUsers">
      <attribute name="title">
       <string>Users</string>
      </attribute>
      <layout class="QGridLayout" name="layUsers">
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <property name="spacing">
        <number>12</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblDefaultPath">
         <property name="text">
          <string>Default $PATH for logged in users</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="txtDefaultPath"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblListShell">
         <property name="text">
          <string>Comma-separated list of shells</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLineEdit" name="txtHideShells"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="lblUserHide">
         <property name="text">
          <string>Comma-separated list of users that should not be listed</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLineEdit" name="txtHideUsers"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblUidMin">
         <property name="text">
          <string>Minimum user id for displayed users</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="sbxUidMin">
         <property name="minimum">
          <number>1000</number>
         </property>
         <property name="maximum">
          <number>60513</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblUidMax">
         <property name="text">
          <string>Maximum user id for displayed users</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="sbxUidMax">
         <property name="minimum">
          <number>1000</number>
         </property>
         <property name="maximum">
          <number>60513</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QCheckBox" name="chkRememberLastSession">
         <property name="text">
          <string>Remember the session of the last successfully logged in user</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QCheckBox" name="chkRememberLastUser">
         <property name="text">
          <string>Remember the last successfully logged in user</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWayland">
      <attribute name="title">
       <string>Wayland</string>
      </attribute>
      <layout class="QGridLayout" name="layWayland">
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <property name="spacing">
        <number>12</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="chkWaylandEnableHiDPI">
         <property name="text">
          <string>Enable Qt's automatic high-DPI scaling</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblWaylandSessionCommand">
         <property name="text">
          <string>Path to a script to execute when starting the desktop session</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="txtWaylandSessionCommand"/>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="tbnWaylandSessionCommand">
         <property name="icon">
          <iconset theme="document-open">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblWaylandSessionLogFile">
         <property name="text">
          <string>Path to the user session log file</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="txtWaylandSessionLogFile"/>
       </item>
       <item row="4" column="1">
        <widget class="QToolButton" name="tbnWaylandSessionLogFile">
         <property name="icon">
          <iconset theme="document-open">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblWaylandSessionDir">
         <property name="text">
          <string>Directory containing available Wayland sessions</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLineEdit" name="txtWaylandSessionDir"/>
       </item>
       <item row="6" column="1">
        <widget class="QToolButton" name="tbnWaylandSessionDir">
         <property name="icon">
          <iconset theme="folder-open">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <spacer name="vspWayland">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabX11">
      <attribute name="title">
       <string>X11</string>
      </attribute>
      <layout class="QVBoxLayout" name="layX11">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>520</width>
            <height>613</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="3">
            <widget class="QCheckBox" name="chkX11EnableHiDPI">
             <property name="text">
              <string>Enable Qt's automatic high-DPI scaling</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lblX11MinimumVT">
             <property name="text">
              <string>The lowest virtual terminal number that will be used</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QToolButton" name="tbnX11SessionDir">
             <property name="icon">
              <iconset theme="folder-open">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item row="8" column="2">
            <widget class="QToolButton" name="tbnX11DisplayCommand">
             <property name="icon">
              <iconset theme="document-open">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item row="10" column="2">
            <widget class="QToolButton" name="tbnX11DisplayStopCommand">
             <property name="icon">
              <iconset theme="document-open">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item row="12" column="2">
            <widget class="QToolButton" name="tbnX11SessionCommand">
             <property name="icon">
              <iconset theme="document-open">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item row="14" column="2">
            <widget class="QToolButton" name="tbnX11ServerPath">
             <property name="icon">
              <iconset theme="document-open">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item row="16" column="2">
            <widget class="QToolButton" name="tbnXauthPath">
             <property name="icon">
              <iconset theme="document-open">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item row="18" column="2">
            <widget class="QToolButton" name="tbnXephyrPath">
             <property name="icon">
              <iconset theme="document-open">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item row="20" column="2">
            <widget class="QToolButton" name="tbnX11UserAuthFile">
             <property name="icon">
              <iconset theme="document-open">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item row="22" column="2">
            <widget class="QToolButton" name="tbnX11SessionLogFile">
             <property name="icon">
              <iconset theme="document-open">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QLineEdit" name="txtX11SessionDir"/>
           </item>
           <item row="4" column="0" colspan="3">
            <widget class="QLineEdit" name="txtX11ServerArguments"/>
           </item>
           <item row="1" column="1" rowspan="2" colspan="2">
            <widget class="QSpinBox" name="sbxX11MinimumVT"/>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QLineEdit" name="txtX11DisplayCommand"/>
           </item>
           <item row="10" column="0" colspan="2">
            <widget class="QLineEdit" name="txtX11DisplayStopCommand"/>
           </item>
           <item row="12" column="0" colspan="2">
            <widget class="QLineEdit" name="txtX11SessionCommand"/>
           </item>
           <item row="13" column="0" colspan="3">
            <widget class="QLabel" name="lblX11ServerPath">
             <property name="text">
              <string>Path to X server binary</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="3">
            <widget class="QLabel" name="lblX11SessionCommand">
             <property name="text">
              <string>Path to a script to execute when starting the desktop session</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QLabel" name="lblX11ServerArguments">
             <property name="text">
              <string>Arguments passed to the X server invocation</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="3">
            <widget class="QLabel" name="lblX11SessionDir">
             <property name="text">
              <string>Directory containing available X sessions</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="3">
            <widget class="QLabel" name="lblX11DisplayCommand">
             <property name="text">
              <string>Path to a script to execute when starting the display server</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="3">
            <widget class="QLabel" name="lblX11DisplayStopCommand">
             <property name="text">
              <string>Path to a script to execute when stopping the display server</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0" colspan="2">
            <widget class="QLineEdit" name="txtX11ServerPath"/>
           </item>
           <item row="15" column="0" colspan="3">
            <widget class="QLabel" name="lblXauthPath">
             <property name="text">
              <string>Path to xauth binary</string>
             </property>
            </widget>
           </item>
           <item row="17" column="0" colspan="3">
            <widget class="QLabel" name="lblXephyrPath">
             <property name="text">
              <string>Path to Xephyr binary</string>
             </property>
            </widget>
           </item>
           <item row="16" column="0" colspan="2">
            <widget class="QLineEdit" name="txtXauthPath"/>
           </item>
           <item row="18" column="0" colspan="2">
            <widget class="QLineEdit" name="txtXephyrPath"/>
           </item>
           <item row="20" column="0" colspan="2">
            <widget class="QLineEdit" name="txtX11UserAuthFile"/>
           </item>
           <item row="22" column="0" colspan="2">
            <widget class="QLineEdit" name="txtX11SessionLogFile"/>
           </item>
           <item row="21" column="0" colspan="3">
            <widget class="QLabel" name="lblX11SessionLogFile">
             <property name="text">
              <string>Path to the user session log file</string>
             </property>
            </widget>
           </item>
           <item row="19" column="0" colspan="3">
            <widget class="QLabel" name="lblX11UserAuthFile">
             <property name="text">
              <string>Path to the Xauthority file</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFile">
      <attribute name="title">
       <string>File</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="txtFile">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layButtons">
     <item>
      <widget class="QPushButton" name="pbnAbout">
       <property name="text">
        <string>About</string>
       </property>
       <property name="icon">
        <iconset theme="help-about">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
